dotcom-rendering/src/web/components/SignInGate/types.ts

```diff
@@ -18,15 +19,54 @@ export type SignInGateComponent = {
 	}: CanShowGateProps) => Promise<boolean>;
 };
 
+export const ALL_USER_TYPES = ['new', 'guest', 'current'] as const;
+type UserTypeTuple = typeof ALL_USER_TYPES;
+export type UserType = UserTypeTuple[number];
+
+export function isUserType(value: unknown): value is UserType {
```

Comment:
There's a couple of places in this file where we use `function` definitions, typically in DCR I've seen functions defined with the arrow syntax (e.g `export const isUserType = (value: unknown): value isUserType => ALL_USER_TYPES.includes(value as UserType);`) so I'd probably recommend refactoring to use arrow functions :)

Reply:
Ah thanks for the heads up, yep can make that change.